.github/workflows/test-lint-workflow.yml

```diff
@@ -26,3 +26,11 @@ jobs:
       - uses: ./lint-workflow
         with:
           workflows: ${{ steps.workflow-list.outputs.workflow-list }}
+
+      - name: Run Tests
+        run: |
+          echo "This will fail"
+
+      - name: Test 2
+        run: >
+          echo "This will not fail too"
```

Comment:
🌱 It would be nice to define these step's names to articulate what the commands are doing. This would help users when they review the output of the steps in the GitHub action panel.

Reply:
I have updated the step names 👍🏽 @mimartin12